assets/js/guides.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function () {
+
+  //Dismiss cta on guides page
+  $('.dismiss-cta').on('click', (e) => {
+    e.preventDefault();
+    $('.guides-cta-card').hide();
+  })
+
+  //Validate email
+  let validateEmail = (email) => {
+    let re = /^([a-zA-Z0-9_.-])+@(([a-zA-Z0-9-])+.)+([a-zA-Z0-9]{2,4})+$/;
+    return re.test(email);
+  }
+
+  $('#email').focus(() => {
+    $('#newsletter-form input').css("border", "none");
+  })
```

Comment:
Seems like this could be done in pure CSS

Reply:
(another one that could go in a new newsletter-only PR)